### PR TITLE
add webstuff files

### PR DESCRIPTION
**Pull request by acolombo1**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            <section id="project">
+                <div class="project-tile">
+                    <a class="mylink" href="https://www.spacex.com/">My project</a>
+                </div>
+            </section>
+            <section id="project2">
+
+            </section>
+        </div>
+        <script src="" async defer></script>
```
Comment (acolombo1) [inline index.html:46]: Please don't leave not functional code, it leads to a not clean code.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]>      <html class="no-js"> <!--<![endif]-->
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title></title>
```
Comment (acolombo1) [inline index.html:10]: Please add a title to your page

**Review by acolombo1 — COMMENTED** [review]
Good use of semantic tags! 💯

Please configure the linters, take a look at https://github.com/microverseinc/curriculum-transversal-skills/blob/main/clean-code/linters.md. 😮
Please add your title and a detailed description in your readme file.👍
Please, don't leave files like "logic.js" that have no function in the repository.👍
Please give a pertinent title and a detailed description in your pull request.👍
Please configure your code editor for 2 spaces indentation. (the linter will also tell you about it)👍

Please submit changes for the items reviewed and ask for a re-review.

File: index.html (hunk 2 above)
Comment (xtrahuman) [reply index.html:10]: I think the developer would appreciate it if you explained why adding a title on the page is necessary, so you could elaborate further on this comment. 👍

Comment (acolombo1) [reply index.html:10]: It is important to provide a title value that describes the page's purpose. It is shown in the browser's corresponding tab and is useful for assistive technology readers. :heavy_check_mark:

**Review by xtrahuman — CHANGES_REQUESTED** [review]
# Required Changes ♻️

Hello @acolombo1, 

Great first attempt, you have the potential to be a great reviewer

## Highlights
- [x] Great job reviewing your project
- [x] Proper use of inline comments  

image

There are some little issues stopping this project from getting approved.

- [x] Kindly make sure you have only one review summary at the end of your review, please delete one.
- [x] please make sure all of your feedback that involves errors in a code or file are inline comments. The final review comment should be used to summarize the review only.
- [x] Add gif and emojis to make the review look fun 😄 👯 

_Kindly check the comments in your review._ ⬆️  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**